internal/kafka/command_broker_describe.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func (c *brokerCommand) newDescribeCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "describe [id]",
+		Use:   "describe <id>",
```

Comment:
Do we want to call this `confluent kafka broker configuration describe`?

Reply:
I was working on it on this branch but I'll merge this one and open a followup.